trollmoves/dispatcher.py

```diff
@@ -318,7 +321,19 @@ def run(self):
                     continue
                 destinations = self.get_destinations(msg)
                 if destinations:
-                    success = dispatch(msg.data['uri'], destinations)
+                    # Check if the url are on another host:
+                    url = urlparse(msg.data['uri'])
+                    if url.scheme != '':
+                        netloc = url.netloc
```

Comment:
This is unnecessary

Reply:
Well, probably/strictly you are right, but I don't like this:
```
In [7]: path2 = '//san1/polar_out/direct_readout/viirs_active_fires/filtered/AFIMG_j01_d20200423_t110200.geojson'                                                      

In [8]: url2 = urlparse(path2)                                                                                                                                         

In [10]: url2.netloc                                                                                                                                                   
Out[10]: 'san1'
```
In theory `san1` could be the name of a host?